Added info about OpenSSL requirement to README (#35)
KoffeinFlummi/armake2: README.md

cargo build --release
```

> In order to build, you'll need to have OpenSSL installed on your system.<br>

Reviewer: ```suggestion
In order to build, you'll need to have OpenSSL installed on your system.

```

> On **Linux**, the easiest way is to install OpenSSL via your system's package manager (if it is not installed already). Make sure you also have the development packages of openssl installed. For example, `libssl-dev` on Ubuntu or `openssl-devel` on Fedora.<br>

Reviewer: ```suggestion
On **Linux**, the easiest way is to install OpenSSL via your system's package manager (if it is not installed already). Make sure you also have the development packages of OpenSSL installed. For example, `libssl-dev` on Ubuntu or `openssl-devel` on Fedora.

```

> On **Windows**, the easiest way to get compilation and static linking of OpenSSL to work is to download [precompiled OpenSSL binaries](http://slproweb.com/products/Win32OpenSSL.html) (non-light, 64bit) and set the following environment variables:

Reviewer: ```suggestion
On **Windows**, the easiest way to get compilation and static linking of OpenSSL to work is to download [pre-compiled OpenSSL binaries](http://slproweb.com/products/Win32OpenSSL.html) (non-light, 64-bit) and set the following environment variables:
```
